gridstatus/tests/source_specific/test_ercot_api.py

```diff
@@ -358,15 +358,16 @@ def test_get_as_reports_historical_date_range(self):
 
         self._check_as_reports(df, before_full_columns=True)
 
-    @api_vcr.use_cassette("test_get_as_reports_full_columns_21_days_ago.yaml")
+    # TODO: reneable cassettes when we can filter out the JWT request and response
+    # @api_vcr.use_cassette("test_get_as_reports_full_columns_21_days_ago.yaml")
     def test_get_as_reports_full_columns(self):
         df = self.iso.get_as_reports(
             self.local_start_of_today() - pd.DateOffset(days=21),
         )
 
         self._check_as_reports(df)
 
-    @api_vcr.use_cassette("test_get_as_reports_dst_end_2024_11_03.yaml")
+    # @api_vcr.use_cassette("test_get_as_reports_dst_end_2024_11_03.yaml")
```

Comment:
Do we want to leave them out of the githistory (like with https://github.com/gridstatus/gridstatus/pull/491) so they still work locally? Or are we not confident they won't make their way back to source control?